app/Models/Changelog.php

```diff
@@ -42,8 +54,33 @@ public function scopeListing($query, $offset = 20)
             ->orderBy('changelog_id', 'desc');
     }
 
+    public function scopeDefault($query)
+    {
+        return $query->where('private', 0);
+    }
+
     public function user()
     {
         return $this->belongsTo(User::class, 'user_id');
     }
+
+    public function updateStream()
+    {
+        return $this->hasOne(UpdateStream::class, 'stream_id', 'stream_id');
+    }
+
+    public function gameBuild()
+    {
+        return $this->belongsTo(Build::class, 'build', 'version');
+    }
+
+    public function getPrefixAttribute($value)
+    {
+        return self::ICONS[$value];
```

Comment:
wouldn't it make sense to call `TOOLTIPS` as `PREFIXES` instead and have this as `icon` (attribute, plain function, or even just helper)?